### DISCUSSION TITLE
Trier des journaux provenant de différents automates.

### DISCUSSION THREAD

**Discussion by PaulV54** [witsa/synapps#793] [Q&A] [answered]
Bonjour.

Je travaille actuellement sur un projet utilisant trois automates. J'aimerais regrouper les journaux de chaque automate dans un seul journal, et montrer la provenance de chaque alerte sur le Synapps. J'ai déjà transféré tous les journaux sur un seul automate à l'aide de sites WOP, mais je ne sais pas comment ajouter le site de provenance des alertes dans le journal Synapps, ou si c'est seulement possible.

Une autre idée était de créer trois journaux séparés, un pour chaque automate, mais je n'ai pas trouvé d'option pour afficher uniquement les évènements d'un site spécifique sur un journal.

Ainsi, existe-t-il des options pour réaliser une des deux idées proposées ci-dessus, ou existe-t-il une autre méthode pour montrer la provenance des alertes dans le journal ?

Merci d'avance pour votre temps.

**Comment by manufitoussiwit** [comment, marked as answer]
Bonjour, 
L'acteur *Journal des variables* possède la colonne Site qui doit normalement indiquer l'origine du point.

![image](https://github.com/user-attachments/assets/a96bf7bb-f8ea-4be7-b3ef-2312696869be)

**Reply by PaulV54** [reply]
Bonjour.
Merci pour votre aide, j'ai pu résoudre mon problème.
